flask_website/listings/extensions.py

```diff
@@ -72,10 +72,14 @@ def docserver(self):
     ),
     Extension('Flask-SQLAlchemy', 'Armin Ronacher',
         description='''
-            <p>Adds SQLAlchemy support to Flask.  Quick and easy.
+            <p>Flask-SQLAlchemy is an extension for Flask that adds support
+            for SQLAlchemy to your application. It requires SQLAlchemy 0.6
```

Comment:
- We already know it's an extension for Flask, it's on the Flask website's extension page.
- It requires 0.8 or higher.